packages/api/src/schema/schema.yaml

```diff
@@ -1118,6 +1108,11 @@ components:
           description: ID of the output asset
           index: true
           example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+        outputAssetsIds:
```

Comment:
Same concern regarding the unindexed array 
https://github.com/livepeer/task-runner/pull/7#discussion_r805917093